sktime/markov/msm/markov_state_model.py

```diff
@@ -1213,3 +1221,145 @@ def score(self, dtrajs, score_method='VAMP2', score_k=10):
         from sktime.metrics import vamp_score
         return vamp_score(K, C00_train, C0t_train, Ctt_train, C00_test, C0t_test, Ctt_test,
                           k=score_k, score=score_method)
+
+
+class MarkovStateModelCollection(MarkovStateModel):
+    r"""A collection of Markov state models. An instance of this itself behaves like a
+    :class:`MSM <MarkovStateModel>`, defaulting to the first (transition matrix, stationary distribution,
```

Comment:
does the collection have some knowledge of the percentage of data contained in each set?

Reply:
i guess not since there may not be data? but if there is data is there somewhere else where these kinds of percent occupancies are provided?

Reply:
i.e. if there are also count models

Reply:
implicitly, yes, because each MSM in the collection has a count model attached to it which knows its selected count/state fraction. But this is a bit buried so I guess it might be helpful to offer methods for this on the collection too